### PR TITLE
Correct loss

### PR DESCRIPTION
**Pull request by adammenges**
As seen here:

https://github.com/ericjang/gumbel-softmax/blob/master/Categorical%20VAE.ipynb

`loss=tf.reduce_mean(-elbo)`

### CODE REVIEW

**Comment by EderSantana** [comment]
Thanks, how did the outputs looks like in the notebook. Would you want to update it as well?
